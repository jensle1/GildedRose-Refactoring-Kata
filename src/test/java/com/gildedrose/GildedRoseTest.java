package com.gildedrose;

import org.approvaltests.combinations.CombinationApprovals;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class GildedRoseTest {

    private final String agedBrie = "Aged Brie";
    private final String sulfuras = "Sulfuras, Hand of Ragnaros";
    private final String backstage = "Backstage passes to a TAFKAL80ETC concert";

    @Test
    void foo() {
        String[] names = {"foo", agedBrie, backstage, sulfuras};
        Integer[] qualities = {0, 1, -1, 49, 50, 51, 80};
        Integer[] sellins = {-1, 0, 1, 2, 4, 5, 6, 8, 9, 10, 11};

        CombinationApprovals.verifyAllCombinations(this::doStuff, names, sellins, qualities);
    }

    public String doStuff(String name, Integer sellin, Integer quality) {
        Item item = new Item(name, sellin, quality);
        GildedRose gildedRose = new GildedRose(new Item[]{item});

        gildedRose.updateQuality();

        return gildedRose.items[0].toString();
    }

}
