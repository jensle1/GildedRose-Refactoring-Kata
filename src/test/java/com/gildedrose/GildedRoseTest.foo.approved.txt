[foo, -1, 0] => foo, -2, 0 
[foo, -1, 1] => foo, -2, 0 
[foo, -1, -1] => foo, -2, -1 
[foo, -1, 49] => foo, -2, 47 
[foo, -1, 50] => foo, -2, 48 
[foo, -1, 51] => foo, -2, 49 
[foo, -1, 80] => foo, -2, 78 
[foo, 0, 0] => foo, -1, 0 
[foo, 0, 1] => foo, -1, 0 
[foo, 0, -1] => foo, -1, -1 
[foo, 0, 49] => foo, -1, 47 
[foo, 0, 50] => foo, -1, 48 
[foo, 0, 51] => foo, -1, 49 
[foo, 0, 80] => foo, -1, 78 
[foo, 1, 0] => foo, 0, 0 
[foo, 1, 1] => foo, 0, 0 
[foo, 1, -1] => foo, 0, -1 
[foo, 1, 49] => foo, 0, 48 
[foo, 1, 50] => foo, 0, 49 
[foo, 1, 51] => foo, 0, 50 
[foo, 1, 80] => foo, 0, 79 
[foo, 2, 0] => foo, 1, 0 
[foo, 2, 1] => foo, 1, 0 
[foo, 2, -1] => foo, 1, -1 
[foo, 2, 49] => foo, 1, 48 
[foo, 2, 50] => foo, 1, 49 
[foo, 2, 51] => foo, 1, 50 
[foo, 2, 80] => foo, 1, 79 
[foo, 4, 0] => foo, 3, 0 
[foo, 4, 1] => foo, 3, 0 
[foo, 4, -1] => foo, 3, -1 
[foo, 4, 49] => foo, 3, 48 
[foo, 4, 50] => foo, 3, 49 
[foo, 4, 51] => foo, 3, 50 
[foo, 4, 80] => foo, 3, 79 
[foo, 5, 0] => foo, 4, 0 
[foo, 5, 1] => foo, 4, 0 
[foo, 5, -1] => foo, 4, -1 
[foo, 5, 49] => foo, 4, 48 
[foo, 5, 50] => foo, 4, 49 
[foo, 5, 51] => foo, 4, 50 
[foo, 5, 80] => foo, 4, 79 
[foo, 6, 0] => foo, 5, 0 
[foo, 6, 1] => foo, 5, 0 
[foo, 6, -1] => foo, 5, -1 
[foo, 6, 49] => foo, 5, 48 
[foo, 6, 50] => foo, 5, 49 
[foo, 6, 51] => foo, 5, 50 
[foo, 6, 80] => foo, 5, 79 
[foo, 8, 0] => foo, 7, 0 
[foo, 8, 1] => foo, 7, 0 
[foo, 8, -1] => foo, 7, -1 
[foo, 8, 49] => foo, 7, 48 
[foo, 8, 50] => foo, 7, 49 
[foo, 8, 51] => foo, 7, 50 
[foo, 8, 80] => foo, 7, 79 
[foo, 9, 0] => foo, 8, 0 
[foo, 9, 1] => foo, 8, 0 
[foo, 9, -1] => foo, 8, -1 
[foo, 9, 49] => foo, 8, 48 
[foo, 9, 50] => foo, 8, 49 
[foo, 9, 51] => foo, 8, 50 
[foo, 9, 80] => foo, 8, 79 
[foo, 10, 0] => foo, 9, 0 
[foo, 10, 1] => foo, 9, 0 
[foo, 10, -1] => foo, 9, -1 
[foo, 10, 49] => foo, 9, 48 
[foo, 10, 50] => foo, 9, 49 
[foo, 10, 51] => foo, 9, 50 
[foo, 10, 80] => foo, 9, 79 
[foo, 11, 0] => foo, 10, 0 
[foo, 11, 1] => foo, 10, 0 
[foo, 11, -1] => foo, 10, -1 
[foo, 11, 49] => foo, 10, 48 
[foo, 11, 50] => foo, 10, 49 
[foo, 11, 51] => foo, 10, 50 
[foo, 11, 80] => foo, 10, 79 
[Aged Brie, -1, 0] => Aged Brie, -2, 2 
[Aged Brie, -1, 1] => Aged Brie, -2, 3 
[Aged Brie, -1, -1] => Aged Brie, -2, 1 
[Aged Brie, -1, 49] => Aged Brie, -2, 50 
[Aged Brie, -1, 50] => Aged Brie, -2, 50 
[Aged Brie, -1, 51] => Aged Brie, -2, 51 
[Aged Brie, -1, 80] => Aged Brie, -2, 80 
[Aged Brie, 0, 0] => Aged Brie, -1, 2 
[Aged Brie, 0, 1] => Aged Brie, -1, 3 
[Aged Brie, 0, -1] => Aged Brie, -1, 1 
[Aged Brie, 0, 49] => Aged Brie, -1, 50 
[Aged Brie, 0, 50] => Aged Brie, -1, 50 
[Aged Brie, 0, 51] => Aged Brie, -1, 51 
[Aged Brie, 0, 80] => Aged Brie, -1, 80 
[Aged Brie, 1, 0] => Aged Brie, 0, 1 
[Aged Brie, 1, 1] => Aged Brie, 0, 2 
[Aged Brie, 1, -1] => Aged Brie, 0, 0 
[Aged Brie, 1, 49] => Aged Brie, 0, 50 
[Aged Brie, 1, 50] => Aged Brie, 0, 50 
[Aged Brie, 1, 51] => Aged Brie, 0, 51 
[Aged Brie, 1, 80] => Aged Brie, 0, 80 
[Aged Brie, 2, 0] => Aged Brie, 1, 1 
[Aged Brie, 2, 1] => Aged Brie, 1, 2 
[Aged Brie, 2, -1] => Aged Brie, 1, 0 
[Aged Brie, 2, 49] => Aged Brie, 1, 50 
[Aged Brie, 2, 50] => Aged Brie, 1, 50 
[Aged Brie, 2, 51] => Aged Brie, 1, 51 
[Aged Brie, 2, 80] => Aged Brie, 1, 80 
[Aged Brie, 4, 0] => Aged Brie, 3, 1 
[Aged Brie, 4, 1] => Aged Brie, 3, 2 
[Aged Brie, 4, -1] => Aged Brie, 3, 0 
[Aged Brie, 4, 49] => Aged Brie, 3, 50 
[Aged Brie, 4, 50] => Aged Brie, 3, 50 
[Aged Brie, 4, 51] => Aged Brie, 3, 51 
[Aged Brie, 4, 80] => Aged Brie, 3, 80 
[Aged Brie, 5, 0] => Aged Brie, 4, 1 
[Aged Brie, 5, 1] => Aged Brie, 4, 2 
[Aged Brie, 5, -1] => Aged Brie, 4, 0 
[Aged Brie, 5, 49] => Aged Brie, 4, 50 
[Aged Brie, 5, 50] => Aged Brie, 4, 50 
[Aged Brie, 5, 51] => Aged Brie, 4, 51 
[Aged Brie, 5, 80] => Aged Brie, 4, 80 
[Aged Brie, 6, 0] => Aged Brie, 5, 1 
[Aged Brie, 6, 1] => Aged Brie, 5, 2 
[Aged Brie, 6, -1] => Aged Brie, 5, 0 
[Aged Brie, 6, 49] => Aged Brie, 5, 50 
[Aged Brie, 6, 50] => Aged Brie, 5, 50 
[Aged Brie, 6, 51] => Aged Brie, 5, 51 
[Aged Brie, 6, 80] => Aged Brie, 5, 80 
[Aged Brie, 8, 0] => Aged Brie, 7, 1 
[Aged Brie, 8, 1] => Aged Brie, 7, 2 
[Aged Brie, 8, -1] => Aged Brie, 7, 0 
[Aged Brie, 8, 49] => Aged Brie, 7, 50 
[Aged Brie, 8, 50] => Aged Brie, 7, 50 
[Aged Brie, 8, 51] => Aged Brie, 7, 51 
[Aged Brie, 8, 80] => Aged Brie, 7, 80 
[Aged Brie, 9, 0] => Aged Brie, 8, 1 
[Aged Brie, 9, 1] => Aged Brie, 8, 2 
[Aged Brie, 9, -1] => Aged Brie, 8, 0 
[Aged Brie, 9, 49] => Aged Brie, 8, 50 
[Aged Brie, 9, 50] => Aged Brie, 8, 50 
[Aged Brie, 9, 51] => Aged Brie, 8, 51 
[Aged Brie, 9, 80] => Aged Brie, 8, 80 
[Aged Brie, 10, 0] => Aged Brie, 9, 1 
[Aged Brie, 10, 1] => Aged Brie, 9, 2 
[Aged Brie, 10, -1] => Aged Brie, 9, 0 
[Aged Brie, 10, 49] => Aged Brie, 9, 50 
[Aged Brie, 10, 50] => Aged Brie, 9, 50 
[Aged Brie, 10, 51] => Aged Brie, 9, 51 
[Aged Brie, 10, 80] => Aged Brie, 9, 80 
[Aged Brie, 11, 0] => Aged Brie, 10, 1 
[Aged Brie, 11, 1] => Aged Brie, 10, 2 
[Aged Brie, 11, -1] => Aged Brie, 10, 0 
[Aged Brie, 11, 49] => Aged Brie, 10, 50 
[Aged Brie, 11, 50] => Aged Brie, 10, 50 
[Aged Brie, 11, 51] => Aged Brie, 10, 51 
[Aged Brie, 11, 80] => Aged Brie, 10, 80 
[Backstage passes to a TAFKAL80ETC concert, -1, 0] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 1] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, -1] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 49] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 50] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 51] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, -1, 80] => Backstage passes to a TAFKAL80ETC concert, -2, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 0] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 1] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, -1] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 49] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 50] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 51] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 0, 80] => Backstage passes to a TAFKAL80ETC concert, -1, 0 
[Backstage passes to a TAFKAL80ETC concert, 1, 0] => Backstage passes to a TAFKAL80ETC concert, 0, 3 
[Backstage passes to a TAFKAL80ETC concert, 1, 1] => Backstage passes to a TAFKAL80ETC concert, 0, 4 
[Backstage passes to a TAFKAL80ETC concert, 1, -1] => Backstage passes to a TAFKAL80ETC concert, 0, 2 
[Backstage passes to a TAFKAL80ETC concert, 1, 49] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 1, 50] => Backstage passes to a TAFKAL80ETC concert, 0, 50 
[Backstage passes to a TAFKAL80ETC concert, 1, 51] => Backstage passes to a TAFKAL80ETC concert, 0, 51 
[Backstage passes to a TAFKAL80ETC concert, 1, 80] => Backstage passes to a TAFKAL80ETC concert, 0, 80 
[Backstage passes to a TAFKAL80ETC concert, 2, 0] => Backstage passes to a TAFKAL80ETC concert, 1, 3 
[Backstage passes to a TAFKAL80ETC concert, 2, 1] => Backstage passes to a TAFKAL80ETC concert, 1, 4 
[Backstage passes to a TAFKAL80ETC concert, 2, -1] => Backstage passes to a TAFKAL80ETC concert, 1, 2 
[Backstage passes to a TAFKAL80ETC concert, 2, 49] => Backstage passes to a TAFKAL80ETC concert, 1, 50 
[Backstage passes to a TAFKAL80ETC concert, 2, 50] => Backstage passes to a TAFKAL80ETC concert, 1, 50 
[Backstage passes to a TAFKAL80ETC concert, 2, 51] => Backstage passes to a TAFKAL80ETC concert, 1, 51 
[Backstage passes to a TAFKAL80ETC concert, 2, 80] => Backstage passes to a TAFKAL80ETC concert, 1, 80 
[Backstage passes to a TAFKAL80ETC concert, 4, 0] => Backstage passes to a TAFKAL80ETC concert, 3, 3 
[Backstage passes to a TAFKAL80ETC concert, 4, 1] => Backstage passes to a TAFKAL80ETC concert, 3, 4 
[Backstage passes to a TAFKAL80ETC concert, 4, -1] => Backstage passes to a TAFKAL80ETC concert, 3, 2 
[Backstage passes to a TAFKAL80ETC concert, 4, 49] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 50] => Backstage passes to a TAFKAL80ETC concert, 3, 50 
[Backstage passes to a TAFKAL80ETC concert, 4, 51] => Backstage passes to a TAFKAL80ETC concert, 3, 51 
[Backstage passes to a TAFKAL80ETC concert, 4, 80] => Backstage passes to a TAFKAL80ETC concert, 3, 80 
[Backstage passes to a TAFKAL80ETC concert, 5, 0] => Backstage passes to a TAFKAL80ETC concert, 4, 3 
[Backstage passes to a TAFKAL80ETC concert, 5, 1] => Backstage passes to a TAFKAL80ETC concert, 4, 4 
[Backstage passes to a TAFKAL80ETC concert, 5, -1] => Backstage passes to a TAFKAL80ETC concert, 4, 2 
[Backstage passes to a TAFKAL80ETC concert, 5, 49] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 50] => Backstage passes to a TAFKAL80ETC concert, 4, 50 
[Backstage passes to a TAFKAL80ETC concert, 5, 51] => Backstage passes to a TAFKAL80ETC concert, 4, 51 
[Backstage passes to a TAFKAL80ETC concert, 5, 80] => Backstage passes to a TAFKAL80ETC concert, 4, 80 
[Backstage passes to a TAFKAL80ETC concert, 6, 0] => Backstage passes to a TAFKAL80ETC concert, 5, 2 
[Backstage passes to a TAFKAL80ETC concert, 6, 1] => Backstage passes to a TAFKAL80ETC concert, 5, 3 
[Backstage passes to a TAFKAL80ETC concert, 6, -1] => Backstage passes to a TAFKAL80ETC concert, 5, 1 
[Backstage passes to a TAFKAL80ETC concert, 6, 49] => Backstage passes to a TAFKAL80ETC concert, 5, 50 
[Backstage passes to a TAFKAL80ETC concert, 6, 50] => Backstage passes to a TAFKAL80ETC concert, 5, 50 
[Backstage passes to a TAFKAL80ETC concert, 6, 51] => Backstage passes to a TAFKAL80ETC concert, 5, 51 
[Backstage passes to a TAFKAL80ETC concert, 6, 80] => Backstage passes to a TAFKAL80ETC concert, 5, 80 
[Backstage passes to a TAFKAL80ETC concert, 8, 0] => Backstage passes to a TAFKAL80ETC concert, 7, 2 
[Backstage passes to a TAFKAL80ETC concert, 8, 1] => Backstage passes to a TAFKAL80ETC concert, 7, 3 
[Backstage passes to a TAFKAL80ETC concert, 8, -1] => Backstage passes to a TAFKAL80ETC concert, 7, 1 
[Backstage passes to a TAFKAL80ETC concert, 8, 49] => Backstage passes to a TAFKAL80ETC concert, 7, 50 
[Backstage passes to a TAFKAL80ETC concert, 8, 50] => Backstage passes to a TAFKAL80ETC concert, 7, 50 
[Backstage passes to a TAFKAL80ETC concert, 8, 51] => Backstage passes to a TAFKAL80ETC concert, 7, 51 
[Backstage passes to a TAFKAL80ETC concert, 8, 80] => Backstage passes to a TAFKAL80ETC concert, 7, 80 
[Backstage passes to a TAFKAL80ETC concert, 9, 0] => Backstage passes to a TAFKAL80ETC concert, 8, 2 
[Backstage passes to a TAFKAL80ETC concert, 9, 1] => Backstage passes to a TAFKAL80ETC concert, 8, 3 
[Backstage passes to a TAFKAL80ETC concert, 9, -1] => Backstage passes to a TAFKAL80ETC concert, 8, 1 
[Backstage passes to a TAFKAL80ETC concert, 9, 49] => Backstage passes to a TAFKAL80ETC concert, 8, 50 
[Backstage passes to a TAFKAL80ETC concert, 9, 50] => Backstage passes to a TAFKAL80ETC concert, 8, 50 
[Backstage passes to a TAFKAL80ETC concert, 9, 51] => Backstage passes to a TAFKAL80ETC concert, 8, 51 
[Backstage passes to a TAFKAL80ETC concert, 9, 80] => Backstage passes to a TAFKAL80ETC concert, 8, 80 
[Backstage passes to a TAFKAL80ETC concert, 10, 0] => Backstage passes to a TAFKAL80ETC concert, 9, 2 
[Backstage passes to a TAFKAL80ETC concert, 10, 1] => Backstage passes to a TAFKAL80ETC concert, 9, 3 
[Backstage passes to a TAFKAL80ETC concert, 10, -1] => Backstage passes to a TAFKAL80ETC concert, 9, 1 
[Backstage passes to a TAFKAL80ETC concert, 10, 49] => Backstage passes to a TAFKAL80ETC concert, 9, 50 
[Backstage passes to a TAFKAL80ETC concert, 10, 50] => Backstage passes to a TAFKAL80ETC concert, 9, 50 
[Backstage passes to a TAFKAL80ETC concert, 10, 51] => Backstage passes to a TAFKAL80ETC concert, 9, 51 
[Backstage passes to a TAFKAL80ETC concert, 10, 80] => Backstage passes to a TAFKAL80ETC concert, 9, 80 
[Backstage passes to a TAFKAL80ETC concert, 11, 0] => Backstage passes to a TAFKAL80ETC concert, 10, 1 
[Backstage passes to a TAFKAL80ETC concert, 11, 1] => Backstage passes to a TAFKAL80ETC concert, 10, 2 
[Backstage passes to a TAFKAL80ETC concert, 11, -1] => Backstage passes to a TAFKAL80ETC concert, 10, 0 
[Backstage passes to a TAFKAL80ETC concert, 11, 49] => Backstage passes to a TAFKAL80ETC concert, 10, 50 
[Backstage passes to a TAFKAL80ETC concert, 11, 50] => Backstage passes to a TAFKAL80ETC concert, 10, 50 
[Backstage passes to a TAFKAL80ETC concert, 11, 51] => Backstage passes to a TAFKAL80ETC concert, 10, 51 
[Backstage passes to a TAFKAL80ETC concert, 11, 80] => Backstage passes to a TAFKAL80ETC concert, 10, 80 
[Sulfuras, Hand of Ragnaros, -1, 0] => Sulfuras, Hand of Ragnaros, -1, 0 
[Sulfuras, Hand of Ragnaros, -1, 1] => Sulfuras, Hand of Ragnaros, -1, 1 
[Sulfuras, Hand of Ragnaros, -1, -1] => Sulfuras, Hand of Ragnaros, -1, -1 
[Sulfuras, Hand of Ragnaros, -1, 49] => Sulfuras, Hand of Ragnaros, -1, 49 
[Sulfuras, Hand of Ragnaros, -1, 50] => Sulfuras, Hand of Ragnaros, -1, 50 
[Sulfuras, Hand of Ragnaros, -1, 51] => Sulfuras, Hand of Ragnaros, -1, 51 
[Sulfuras, Hand of Ragnaros, -1, 80] => Sulfuras, Hand of Ragnaros, -1, 80 
[Sulfuras, Hand of Ragnaros, 0, 0] => Sulfuras, Hand of Ragnaros, 0, 0 
[Sulfuras, Hand of Ragnaros, 0, 1] => Sulfuras, Hand of Ragnaros, 0, 1 
[Sulfuras, Hand of Ragnaros, 0, -1] => Sulfuras, Hand of Ragnaros, 0, -1 
[Sulfuras, Hand of Ragnaros, 0, 49] => Sulfuras, Hand of Ragnaros, 0, 49 
[Sulfuras, Hand of Ragnaros, 0, 50] => Sulfuras, Hand of Ragnaros, 0, 50 
[Sulfuras, Hand of Ragnaros, 0, 51] => Sulfuras, Hand of Ragnaros, 0, 51 
[Sulfuras, Hand of Ragnaros, 0, 80] => Sulfuras, Hand of Ragnaros, 0, 80 
[Sulfuras, Hand of Ragnaros, 1, 0] => Sulfuras, Hand of Ragnaros, 1, 0 
[Sulfuras, Hand of Ragnaros, 1, 1] => Sulfuras, Hand of Ragnaros, 1, 1 
[Sulfuras, Hand of Ragnaros, 1, -1] => Sulfuras, Hand of Ragnaros, 1, -1 
[Sulfuras, Hand of Ragnaros, 1, 49] => Sulfuras, Hand of Ragnaros, 1, 49 
[Sulfuras, Hand of Ragnaros, 1, 50] => Sulfuras, Hand of Ragnaros, 1, 50 
[Sulfuras, Hand of Ragnaros, 1, 51] => Sulfuras, Hand of Ragnaros, 1, 51 
[Sulfuras, Hand of Ragnaros, 1, 80] => Sulfuras, Hand of Ragnaros, 1, 80 
[Sulfuras, Hand of Ragnaros, 2, 0] => Sulfuras, Hand of Ragnaros, 2, 0 
[Sulfuras, Hand of Ragnaros, 2, 1] => Sulfuras, Hand of Ragnaros, 2, 1 
[Sulfuras, Hand of Ragnaros, 2, -1] => Sulfuras, Hand of Ragnaros, 2, -1 
[Sulfuras, Hand of Ragnaros, 2, 49] => Sulfuras, Hand of Ragnaros, 2, 49 
[Sulfuras, Hand of Ragnaros, 2, 50] => Sulfuras, Hand of Ragnaros, 2, 50 
[Sulfuras, Hand of Ragnaros, 2, 51] => Sulfuras, Hand of Ragnaros, 2, 51 
[Sulfuras, Hand of Ragnaros, 2, 80] => Sulfuras, Hand of Ragnaros, 2, 80 
[Sulfuras, Hand of Ragnaros, 4, 0] => Sulfuras, Hand of Ragnaros, 4, 0 
[Sulfuras, Hand of Ragnaros, 4, 1] => Sulfuras, Hand of Ragnaros, 4, 1 
[Sulfuras, Hand of Ragnaros, 4, -1] => Sulfuras, Hand of Ragnaros, 4, -1 
[Sulfuras, Hand of Ragnaros, 4, 49] => Sulfuras, Hand of Ragnaros, 4, 49 
[Sulfuras, Hand of Ragnaros, 4, 50] => Sulfuras, Hand of Ragnaros, 4, 50 
[Sulfuras, Hand of Ragnaros, 4, 51] => Sulfuras, Hand of Ragnaros, 4, 51 
[Sulfuras, Hand of Ragnaros, 4, 80] => Sulfuras, Hand of Ragnaros, 4, 80 
[Sulfuras, Hand of Ragnaros, 5, 0] => Sulfuras, Hand of Ragnaros, 5, 0 
[Sulfuras, Hand of Ragnaros, 5, 1] => Sulfuras, Hand of Ragnaros, 5, 1 
[Sulfuras, Hand of Ragnaros, 5, -1] => Sulfuras, Hand of Ragnaros, 5, -1 
[Sulfuras, Hand of Ragnaros, 5, 49] => Sulfuras, Hand of Ragnaros, 5, 49 
[Sulfuras, Hand of Ragnaros, 5, 50] => Sulfuras, Hand of Ragnaros, 5, 50 
[Sulfuras, Hand of Ragnaros, 5, 51] => Sulfuras, Hand of Ragnaros, 5, 51 
[Sulfuras, Hand of Ragnaros, 5, 80] => Sulfuras, Hand of Ragnaros, 5, 80 
[Sulfuras, Hand of Ragnaros, 6, 0] => Sulfuras, Hand of Ragnaros, 6, 0 
[Sulfuras, Hand of Ragnaros, 6, 1] => Sulfuras, Hand of Ragnaros, 6, 1 
[Sulfuras, Hand of Ragnaros, 6, -1] => Sulfuras, Hand of Ragnaros, 6, -1 
[Sulfuras, Hand of Ragnaros, 6, 49] => Sulfuras, Hand of Ragnaros, 6, 49 
[Sulfuras, Hand of Ragnaros, 6, 50] => Sulfuras, Hand of Ragnaros, 6, 50 
[Sulfuras, Hand of Ragnaros, 6, 51] => Sulfuras, Hand of Ragnaros, 6, 51 
[Sulfuras, Hand of Ragnaros, 6, 80] => Sulfuras, Hand of Ragnaros, 6, 80 
[Sulfuras, Hand of Ragnaros, 8, 0] => Sulfuras, Hand of Ragnaros, 8, 0 
[Sulfuras, Hand of Ragnaros, 8, 1] => Sulfuras, Hand of Ragnaros, 8, 1 
[Sulfuras, Hand of Ragnaros, 8, -1] => Sulfuras, Hand of Ragnaros, 8, -1 
[Sulfuras, Hand of Ragnaros, 8, 49] => Sulfuras, Hand of Ragnaros, 8, 49 
[Sulfuras, Hand of Ragnaros, 8, 50] => Sulfuras, Hand of Ragnaros, 8, 50 
[Sulfuras, Hand of Ragnaros, 8, 51] => Sulfuras, Hand of Ragnaros, 8, 51 
[Sulfuras, Hand of Ragnaros, 8, 80] => Sulfuras, Hand of Ragnaros, 8, 80 
[Sulfuras, Hand of Ragnaros, 9, 0] => Sulfuras, Hand of Ragnaros, 9, 0 
[Sulfuras, Hand of Ragnaros, 9, 1] => Sulfuras, Hand of Ragnaros, 9, 1 
[Sulfuras, Hand of Ragnaros, 9, -1] => Sulfuras, Hand of Ragnaros, 9, -1 
[Sulfuras, Hand of Ragnaros, 9, 49] => Sulfuras, Hand of Ragnaros, 9, 49 
[Sulfuras, Hand of Ragnaros, 9, 50] => Sulfuras, Hand of Ragnaros, 9, 50 
[Sulfuras, Hand of Ragnaros, 9, 51] => Sulfuras, Hand of Ragnaros, 9, 51 
[Sulfuras, Hand of Ragnaros, 9, 80] => Sulfuras, Hand of Ragnaros, 9, 80 
[Sulfuras, Hand of Ragnaros, 10, 0] => Sulfuras, Hand of Ragnaros, 10, 0 
[Sulfuras, Hand of Ragnaros, 10, 1] => Sulfuras, Hand of Ragnaros, 10, 1 
[Sulfuras, Hand of Ragnaros, 10, -1] => Sulfuras, Hand of Ragnaros, 10, -1 
[Sulfuras, Hand of Ragnaros, 10, 49] => Sulfuras, Hand of Ragnaros, 10, 49 
[Sulfuras, Hand of Ragnaros, 10, 50] => Sulfuras, Hand of Ragnaros, 10, 50 
[Sulfuras, Hand of Ragnaros, 10, 51] => Sulfuras, Hand of Ragnaros, 10, 51 
[Sulfuras, Hand of Ragnaros, 10, 80] => Sulfuras, Hand of Ragnaros, 10, 80 
[Sulfuras, Hand of Ragnaros, 11, 0] => Sulfuras, Hand of Ragnaros, 11, 0 
[Sulfuras, Hand of Ragnaros, 11, 1] => Sulfuras, Hand of Ragnaros, 11, 1 
[Sulfuras, Hand of Ragnaros, 11, -1] => Sulfuras, Hand of Ragnaros, 11, -1 
[Sulfuras, Hand of Ragnaros, 11, 49] => Sulfuras, Hand of Ragnaros, 11, 49 
[Sulfuras, Hand of Ragnaros, 11, 50] => Sulfuras, Hand of Ragnaros, 11, 50 
[Sulfuras, Hand of Ragnaros, 11, 51] => Sulfuras, Hand of Ragnaros, 11, 51 
[Sulfuras, Hand of Ragnaros, 11, 80] => Sulfuras, Hand of Ragnaros, 11, 80 
